proto/attestation/reference_value.proto

```diff
@@ -163,7 +163,10 @@ message CBLayerReferenceValues {
 }
 
 message CBReferenceValues {
-  repeated CBLayerReferenceValues layers = 1;
+  RootLayerReferenceValues root_layer = 1;
```

Comment:
I think that CB will also have a kernel layer

Reply:
The kernel layer is captured in CBLayerReferenceValues